GSoC-2025/cellfinder: Implementing a New Deep Learning Architecture (Adithya Sau).md

```diff
@@ -0,0 +1,146 @@
+
+| Week | Tasks | Hours/Week |
+|------|-------|------------|
+| **Community Bonding** | Finalize architecture design, set up test environments, and familiarize with cellfinder's codebase. | 20 |
+| **Week 1–2** | Implement a prototype of the Vision Transformer (ViT) model for 3D classification tasks. | 35 |
+| **Week 3–4** | Integrate the ViT model with cellfinder's preprocessing pipeline and test on sample datasets. | 35 |
+| **Week 5–6** | Develop an active learning framework to reduce manual annotation requirements by enabling uncertainty-based sampling. | 35 |
+| **Week 7–8** | Optimize the model using quantization techniques (e.g., 8-bit models) and add multi-GPU support for large datasets. | 35 |
+| **Week 9–10** | Extend the Napari plugin to enable interactive training/validation workflows using the new model. | 35 |
+| **Week 11–12** | Finalize testing, benchmarking, documentation, and write a blog post summarizing results and challenges. | 35 |
+
+---
+
+### **Communication Plan**
+
+- **Daily Updates:** Share progress updates in the BrainGlobe Zulip stream.
+- **Weekly Meetings:** Schedule video calls with mentors (@IgorTatarnikov and @alessandrofelder) to discuss progress, blockers, and next steps.
+- **PR Reviews:** Submit code regularly via GitHub pull requests for mentor feedback.
+- **Emergency Communication:** Direct messaging via Zulip or email.
+
+---
+
+## **Personal Statement**
+
+I am a third-year undergraduate at BITS Pilani, KK Birla Goa Campus, pursuing a Bachelor's degree in Electronics and Electrical Engineering. With a strong foundation in software engineering, I specialize in Python, machine learning, and building scalable systems.I am a software engineer with expertise in Python, machine learning, and scalable systems. Currently, I am working on a research paper titled "Cognitive Neuroscape: Gamified Frameworks for Multi-Dimensional Cognitive Assessment in Neurodegeneration". This project focuses on designing goal-based games to evaluate cognitive performance across age groups, leveraging machine learning to identify early biomarkers for neurodegenerative diseases.
+
+This aligns closely with the cellfinder project, as both involve applying advanced computational methods to neuroscience challenges. My experience developing scalable AI solutions, such as a multilingual speech recognition app using AWS services and StickerForecast—a market forecasting model ranked in the global top 500—has equipped me with the skills to implement and optimize new architectures for cellfinder. By contributing to this project, I aim to advance open-source tools that empower researchers in neuroscience.
+---
+
+## **Availability**
+
+I am fully available during the GSoC period (25 hours/week). I have no conflicting commitments such as school or work during this time. I may have an mandatory internship from mid-August from my college but I will find 3 hours/day for GSOC work and get the rest of the work done over the weekend.
```

Comment:
Above you state 35 hours per week, can you confirm, and also make sure it fits with the size of the project.